ev3dev2/motor.py

```diff
@@ -1921,6 +1954,98 @@ def on(self, left_speed, right_speed):
         self.left_motor.run_forever()
         self.right_motor.run_forever()
 
+    def follow_line(self,
+            kp, ki, kd,
```

Comment:
Do we have defaults for these?

Reply:
They are so dependent on the individual robot I don't think we could pick defaults that would work well.